### DISCUSSION TITLE
第2节课作业(22班)

### DISCUSSION THREAD

**Discussion by BarryYin** [InternLM/Tutorial#235] [作业]
在这里提交链接

**Comment by Jasimyn** [comment]
https://broadleaf-gemini-274.notion.site/Lecture2-cad4357b53464f6e85f59b86818fac7a

**Comment by lyyilin** [comment]
深夜补作业欢迎大家参观我的博客：https://blog.csdn.net/qq_73668945/article/details/135516255

**Comment by liuwei16** [comment]
https://g5b4uupmby.feishu.cn/wiki/QFEKwUNrfi4bXukLAL9cDwEvnOc

**Comment by Zhiwen-Liu** [comment]
https://github.com/ArchimedesLiu/InternLM-LeaningNote/blob/main/Homework/homework2.md

**Comment by funcfang** [comment]
https://www.yuque.com/funcfang/ke5qbq/uzsisrq2vfu3uwns?singleDoc#%20%E3%80%8A%E4%BD%9C%E4%B8%9A%E3%80%8B

**Comment by lilxmx** [comment]
https://blog.csdn.net/L1142456160/article/details/135491993

**Comment by MinghanJia** [comment]
https://github.com/MinghanJia/InternLM/blob/main/homework2.md

**Comment by aqizhoua** [comment]
https://aqizhoua.github.io/poster/280c916a.html

**Comment by cavedweller509** [comment]
https://github.com/BestAnHongjun/InternLM_Notes/blob/main/homework/homework_2.md

**Comment by cyq2368** [comment]
https://mwsnr0a94gx.feishu.cn/docx/NUzAdHquIoApslxI1mOcLSiunQh?from=from_copylink

**Comment by miyc1996** [comment]
https://github.com/miyc1996/InternLM/blob/main/work1.ipynb

**Comment by tannl** [comment]
https://www.yuque.com/r/note/e5739964-fd23-4ea7-a731-e0cf040bcf2e

**Comment by miku8miku** [comment]
https://note.youdao.com/s/NldhKuQv

**Comment by hyanworkspace** [comment]
https://note.youdao.com/web/#/file/recent/note/WEB595a1bc928d2831aa05de45f91bfaf60/

**Comment by lankuohsing** [comment]
https://github.com/lankuohsing/InternLM_notes/blob/main/lecture_2/homework2.md

**Comment by leonfrank** [comment]
https://github.com/leonfrank/InternLM-HW/blob/main/lecture2/hw2.pdf

**Comment by linkch0** [comment]
https://linkch0.github.io/courses/InternLM/assignment/#l2-demo

**Comment by jiayaozhang** [comment]
https://github.com/jiayaozhang/InternLM_Notes/blob/main/Homework2.md

**Reply by jiayaozhang** [reply]
https://github.com/jiayaozhang/InternLM_Notes/blob/main/Lecture2.md

**Comment by vjsmmk** [comment]
https://github.com/vjsmmk/puyu/blob/main/2/woke

**Comment by Lynn401** [comment]
https://daisy-401.notion.site/Tutorial-2-13305bdb427740ae9fbdc1877050db4f?pvs=4

**Comment by siruzhong** [comment]
https://siru.notion.site/2-Demo-1b00299624aa4f1b8af5739912501d05?pvs=4

Homework is in Section 5

**Comment by CDL0726** [comment]
https://github.com/CDL0726/InternLM_Notes/pull/1

**Comment by TAICHIKF** [comment]
https://github.com/TAICHIKF/InternLM_notes/blob/main/2%20InternLM_Demo.pdf

**Comment by Juanting-Xu** [comment]
https://github.com/Juanting-Xu/InternLM_notes/blob/main/hw2.md

**Comment by GradOpt** [comment]
https://github.com/OswaldHongyu/InternLM_notes/blob/main/homework2.pdf

**Comment by mmb135** [comment]
课后作业(1)https://www.yuque.com/r/note/f27fef69-da7e-4bd7-9435-1eeae242cd1c
课后作业(2) https://www.yuque.com/r/note/9a8c1bf2-1f14-4598-b4cb-567d68a04ec7
课后作业(4)https://www.yuque.com/r/note/fbb15dbf-77cd-49f7-b8d9-263f5a894e46
